com.unity.formats.alembic/Editor/Importer/AlembicImporter.cs

```diff
@@ -49,12 +69,15 @@ public static AssetMoveResult OnWillMoveAsset(string from, string to)
         }
     }
 
-    [ScriptedImporter(5, "abc")]
+    [ScriptedImporter(6, "abc")]
     internal class AlembicImporter : ScriptedImporter
     {
         [SerializeField]
+#pragma warning disable 0649
```

Comment:
This variable is always written by serialization, but the warning would cause packman clean console tests to fail.

Reply:
For future references: https://unity.slack.com/archives/C06TPSM32/p1562336486321800